### PR TITLE
Docs: Fix hello world

### PR DESCRIPTION
**Pull request by ncbaratta**
### Type of change

- Documentation

### Description

Retitle and reconfigure the Hello World example.

### CODE REVIEW

**Comment by ncbaratta** [comment]
@dmuntima how does this look?

File: documentation/book/proc-deploying-example-clients.adoc [hunk 1]
```diff
@@ -1,26 +1,29 @@
...
 .Procedure
 
-You can deploy the client in a single command.
-This is an example command you should substitute _test_ for your topic and _my-cluster-kafka-bootstrap:9092_ by your server address for both producer and consumer.
-
 . Deploy the producer
 +
-`oc run kafka-producer -ti --image={DockerKafka} --restart=Never \-- bin/kafka-console-producer.sh --broker-list _my-cluster-kafka-bootstrap:9092_ --topic _test_`
+[source,subs="+quotes,attributes"]
+oc run kafka-producer -ti --image={DockerKafka} --restart=Never \-- bin/kafka-console-producer.sh --broker-list __<my-producer>__:__<my-port>__ --topic _<my-topic>_
```
Comment (ppatierno) [inline documentation/book/proc-deploying-example-clients.adoc]: Why there is a "my-producer" where we should have the bootstrap service name?

File: documentation/book/proc-deploying-example-clients.adoc [hunk 2]
```diff
@@ -1,26 +1,29 @@
...
+
+. Type your message into the console where the producer is running
 
-. Send messages by writing into the console where you run the producer and hit the enter.
+. Press *Enter* to send the message.
 
 . Deploy the consumer
 +
-`oc run kafka-consumer -ti --image={DockerKafka} --restart=Never \-- bin/kafka-console-consumer.sh --bootstrap-server _my-cluster-kafka-bootstrap:9092_ --topic _test_ --from-beginning`
+[source,subs="+quotes,attributes"]
```
Comment (ppatierno) [inline documentation/book/proc-deploying-example-clients.adoc]: The same as before why "my-consumer" where should he the bootstrap service?

File: documentation/book/proc-deploying-example-clients.adoc (hunk 1 above)
Comment (ncbaratta) [reply documentation/book/proc-deploying-example-clients.adoc]: I was trying to make it styled like the replaceable.  Using that style, what should it say? 

I can change:

__<my-producer>__

TO

__<my-cluster-kafka-bootstrap>__

Does that make sense?

File: documentation/book/proc-deploying-example-clients.adoc (hunk 2 above)
Comment (ncbaratta) [reply documentation/book/proc-deploying-example-clients.adoc]: I was trying to make it styled like the replaceable.  Using that style, what should it say? 

I can change:

__<my-consumer>__

TO

__<my-cluster-kafka-bootstrap>__

Does that make sense?

Comment (scholzj) [reply documentation/book/proc-deploying-example-clients.adoc]: Paolo is right. The bootstrap service should be ``_<cluster-name>-kafka-bootstrap:9092_`.

File: documentation/book/proc-deploying-example-clients.adoc (hunk 1 above)
Comment (scholzj) [reply documentation/book/proc-deploying-example-clients.adoc]: Paolo is right. The bootstrap service should be ``_<cluster-name>-kafka-bootstrap:9092_`.

Comment (ncbaratta) [reply documentation/book/proc-deploying-example-clients.adoc]: Like this?

`__<my-producer>-kafka-bootstrap__:9092`

Or:

`__<my-cluster-kafka-bootstrap>__:9092`

Comment (ncbaratta) [reply documentation/book/proc-deploying-example-clients.adoc]: I think I've got it 

`__<my-cluster>__-kafka-bootstrap:9092`

Comment (scholzj) [reply documentation/book/proc-deploying-example-clients.adoc]: Yes, the last one is the right one

`__<my-cluster>__-kafka-bootstrap:9092`

File: documentation/book/proc-deploying-example-clients.adoc (hunk 2 above)
Comment (ncbaratta) [reply documentation/book/proc-deploying-example-clients.adoc]: All better

**Pull request merged**